resources/assets/uad_lists.json

```
@@ -17363,7 +17363,7 @@
   {
     "id": "com.google.android.googlequicksearchbox",
     "list": "Google",
-    "description": "Google Search box (https://play.google.com/store/apps/details?id=com.google.android.googlequicksearchbox)\nRuns in the background.\nPointless. If you need a shortcut to Google on your homescreen just use a web-browser shortcut. Does also remove the Google Sound Search widget, but you can get that functionality from an app like Shazam, that additionally doesn't run in the background constantly like this package does.",
+    "description": "Google (https://play.google.com/store/apps/details?id=com.google.android.googlequicksearchbox)\nRuns in the background.\nIncludes Google Search Box widget and Google Assistant. If you need a shortcut to Google on your homescreen just use a web-browser shortcut. Does also remove the Google Sound Search widget, but you can get that functionality from an app like Shazam, that additionally doesn't run in the background constantly like this package does.",
```

```suggestion
    "description": "Google app (https://play.google.com/store/apps/details?id=com.google.android.googlequicksearchbox)\nRuns in the background.\nIncludes Google Search Box widget and Google Assistant. If you need a shortcut to Google on your homescreen just use a web-browser shortcut. Does also remove the Google Sound Search widget, but you can get that functionality from an app like Shazam, that additionally doesn't run in the background constantly like this package does.",
```

I think it's the Google app. What do you think of my suggestion? Rest looks good to me.

---

Yeah, I was thinking about that too, but decided not to put app there to be consistent with the other descriptions. On a second look, it sounds much better with 'app' included.

---

Also when you remove this package, it removes functionality for Google Voice input everywhere

---

> Also when you remove this package, it removes functionality for Google Voice input everywhere

Do you think we should add that to the description?

---

Good catch! Yes, I think that should be in the description too.